EzPC/EzPC/tc.ml

```diff
@@ -252,13 +279,19 @@ type sresult = gamma result
 
 let rec check_type_well_formedness (g:gamma) (t:typ) :unit result =
   let bitlen_err (n:int) = Type_error ("Incorrect bitlen, expected: " ^ (string_of_int (Config.get_bitlen ())) ^ ", found: " ^ (string_of_int n), t.metadata) in
+  let check_float_label (l:secret_label) : unit result =
```

Comment:
May be this should be named check_int_label?

Reply:
renamed.